diode-core/shared/src/main/scala-3/diode/package.scala

```diff
@@ -0,0 +1,13 @@
+package diode
```

Comment:
Scala 3 doesn't support package objects so I had to duplicate the code and define these as top-level.